platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -5449,7 +5449,7 @@ DisplayServerX11::DisplayServerX11(const String &p_rendering_driver, WindowMode
 	}
 #else
 #ifdef XKB_ENABLED
-	xkb_loaded = true;
+	bool xkb_loaded = true;
```

Comment:
We probably should set `xkb_loaded_v05p` and `xkb_loaded_v08p` to `true` as well. Current version is 1.5.0, so it's probably reasonable to expect at leas 0.8 when linking system version.

Reply:
Thanks, fixed.